### PR TITLE
Alert Slack when Concourse tasks fail

### PR DESCRIPTION
**Pull request by brucebolt**
What
----

Posts a message to #govuk-corona-services-tech on Slack when Concourse tasks fail.

Example:
Screenshot 2020-05-26 at 10 23 13

Links
-----

Trello card: https://trello.com/c/TSaLQkQp

### CODE REVIEW

**Review by bilbof — APPROVED** [review]
Excellent work 👏

File: concourse/pipeline.yml [hunk 1]
```diff
@@ -111,6 +122,17 @@ jobs:
           GOVUK_NOTIFY_EMAIL_TEMPLATE_ID: d4fddfce-1dfd-4ff5-ab42-156e6ba3e76b
           GOVUK_NOTIFY_SMS_TEMPLATE_ID: a2166a39-91f5-4370-bbbf-c59276975d49
           NOTIFY_API_KEY: ((notify-api-key-stg))
+        on_failure:
+          put: govuk-coronavirus-services-tech-slack
+          params:
+            channel: '#govuk-corona-services-tech'
+            username: 'Concourse (Shield Vulnerable Service)'
```
Comment (issyl0) [inline concourse/pipeline.yml]:
```suggestion
            username: 'Concourse (Shielded Vulnerable Service)'
```

File: concourse/pipeline.yml [hunk 2]
```diff
@@ -128,6 +150,17 @@ jobs:
         file: git-master/concourse/tasks/run-smoke-tests.yml
         params:
           TEST_URL: 'https://gds:((basic-auth-password))@govuk-coronavirus-vulnerable-people-form-stg.cloudapps.digital'
+        on_failure:
+          put: govuk-coronavirus-services-tech-slack
+          params:
+            channel: '#govuk-corona-services-tech'
+            username: 'Concourse (Shield Vulnerable Service)'
```
Comment (issyl0) [inline concourse/pipeline.yml]:
```suggestion
            username: 'Concourse (Shielded Vulnerable Service)'
```

File: concourse/pipeline.yml [hunk 3]
```diff
@@ -153,6 +186,17 @@ jobs:
           GOVUK_NOTIFY_EMAIL_TEMPLATE_ID: eb9f351c-f314-47e4-ba7b-656137521f74
           GOVUK_NOTIFY_SMS_TEMPLATE_ID: 0480c4b5-4e08-4598-b2c1-bbc99990afc2
           NOTIFY_API_KEY: ((notify-api-key-prod))
+        on_failure:
+          put: govuk-coronavirus-services-tech-slack
+          params:
+            channel: '#govuk-corona-services-tech'
+            username: 'Concourse (Shield Vulnerable Service)'
```
Comment (issyl0) [inline concourse/pipeline.yml]:
```suggestion
            username: 'Concourse (Shielded Vulnerable Service)'
```

File: concourse/pipeline.yml [hunk 4]
```diff
@@ -172,3 +216,14 @@ jobs:
         file: git-master/concourse/tasks/run-smoke-tests.yml
         params:
           TEST_URL: 'https://coronavirus-vulnerable-people.service.gov.uk'
+        on_failure:
+          put: govuk-coronavirus-services-tech-slack
+          params:
+            channel: '#govuk-corona-services-tech'
+            username: 'Concourse (Shield Vulnerable Service)'
```
Comment (issyl0) [inline concourse/pipeline.yml]:
```suggestion
            username: 'Concourse (Shielded Vulnerable Service)'
```

File: concourse/pipeline.yml (hunk 1 above)
Comment (brucebolt) [reply concourse/pipeline.yml]: Good spot 👍.  Can't believe I missed this in four places, plus in the screenshot of the actual Slack alert 🤦.

**Pull request merged**